sql-statements/sql-statement-overview.md

```diff
@@ -0,0 +1,300 @@
+| [`CALIBRATE RESOURCE`](/sql-statements/sql-statement-calibrate-resource.md) | Calibrates system resources. |
+| [`CREATE RESOURCE GROUP`](/sql-statements/sql-statement-create-resource-group.md) | Creates a new resource group. |
+| [`DROP RESOURCE GROUP`](/sql-statements/sql-statement-drop-resource-group.md) | Drops a resource group. |
+| [`QUERY WATCH`](/sql-statements/sql-statement-query-watch.md) | Manage runaway query watch list. |
+| [`SET RESOURCE GROUP`](/sql-statements/sql-statement-set-resource-group.md) | Sets a resource group. |
+| [`SHOW CREATE RESOURCE GROUP`](/sql-statements/sql-statement-show-create-resource-group.md) | Shows the CREATE statement for a resource group. |
+
+</CustomContent>
+
+<CustomContent platform="tidb-cloud">
+
+| SQL Statement | Description |
+|---------------|-------------|
+| [`ALTER RESOURCE GROUP`](/sql-statements/sql-statement-alter-resource-group.md) | Modifies a resource group. |
+| [`CREATE RESOURCE GROUP`](/sql-statements/sql-statement-create-resource-group.md) | Creates a new resource group. |
+| [`DROP RESOURCE GROUP`](/sql-statements/sql-statement-drop-resource-group.md) | Drops a resource group. |
+| [`QUERY WATCH`](/sql-statements/sql-statement-query-watch.md) | Manage runaway query watch list. |
+| [`SET RESOURCE GROUP`](/sql-statements/sql-statement-set-resource-group.md) | Sets a resource group. |
+| [`SHOW CREATE RESOURCE GROUP`](/sql-statements/sql-statement-show-create-resource-group.md) | Shows the CREATE statement for a resource group. |
+
+</CustomContent>
+
+## Utility statements
+
+| SQL Statement | Description |
+|---------------|-------------|
+| [`DESC`](/sql-statements/sql-statement-desc.md) | Describes the structure of a table. |
+| [`DESCRIBE`](/sql-statements/sql-statement-describe.md) | Describes the structure of a table. |
+| [`DO`](/sql-statements/sql-statement-do.md) | Executes an expression. |
+| [`EXPLAIN`](/sql-statements/sql-statement-explain.md) | Shows the execution plan of a query. |
+| [`TRACE`](/sql-statements/sql-statement-trace.md) | Traces query execution. |
+| [`USE`](/sql-statements/sql-statement-use.md) | Sets the current database. |
+
+## Show statements
+
+| SQL Statement | Description |
+|---------------|-------------|
+| [`SHOW BUILTINS`](/sql-statements/sql-statement-show-builtins.md) | Lists builtin functions. |
+| [`SHOW CHARACTER SET`](/sql-statements/sql-statement-show-character-set.md) | Lists character sets. |
+| [`SHOW COLLATIONS`](/sql-statements/sql-statement-show-collation.md) | Lists collations. |
+| [`SHOW ERRORS`](/sql-statements/sql-statement-show-errors.md) | Shows error messages. |
+| [`SHOW STATUS`](/sql-statements/sql-statement-show-status.md) | Shows server status. |
+| [`SHOW VARIABLES`](/sql-statements/sql-statement-show-variables.md) | Shows system variables. |
+| [`SHOW WARNINGS`](/sql-statements/sql-statement-show-warnings.md) | Shows warning messages. |
```

Comment:
```suggestion
| [`SHOW WARNINGS`](/sql-statements/sql-statement-show-warnings.md) | Shows warnings from previously executed statements. |
```

Reply:
```
mysql> SELECT 1/0;
+------+
| 1/0  |
+------+
| NULL |
+------+
1 row in set, 1 warning (0.00 sec)

mysql> SHOW WARNINGS;
+---------+------+---------------+
| Level   | Code | Message       |
+---------+------+---------------+
| Warning | 1365 | Division by 0 |
+---------+------+---------------+
1 row in set (0.00 sec)

mysql> ANALYZE TABLE t1;
Query OK, 0 rows affected, 1 warning (0.05 sec)

mysql> SHOW WARNINGS;
+-------+------+--------------------------------------------------------------------------------------------------------------------------------------------------------+
| Level | Code | Message                                                                                                                                                |
+-------+------+--------------------------------------------------------------------------------------------------------------------------------------------------------+
| Note  | 1105 | Analyze use auto adjusted sample rate 1.000000 for table test.t1, reason to use this rate is "TiDB assumes that the table is empty, use sample-rate=1" |
+-------+------+--------------------------------------------------------------------------------------------------------------------------------------------------------+
1 row in set (0.00 sec)
```

I would suggest something like "Shows warnings and notes..."

It is a bit confusing that notes increase the warning count instead of a note count (which doesn't exist). And there is no `SHOW NOTES` or anything similar.